Packages/KeyAppKit/Sources/KeyAppBusiness/Solana/SolanaRealtimeAccount.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  File.swift
+//
+//
+//  Created by Giang Long Tran on 27.04.2023.
+//
+
+import Combine
+import Foundation
+import KeyAppKitCore
+
+/// The service state.
+enum SolanaRealtimeAccountState {
```

Comment:
It is duplicated? `RealtimeSolanaAccountState` and `SolanaRealtimeAccountState`

Reply:
Duplicated